plugins/libvirt/munin-libvirtpy

```diff
@@ -1,57 +1,75 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
-# Revision 1.0  2008/05/16 - Steven Wagner
-# First functional release. Works for me.
-#
-# Revision 0.5  2008/05/01  - Julien Rottenberg
-# initial display of variables from libvirt
+"""
+=encoding utf8
 
-#python-libvirt is required
+=head1 NAME
+
+munin-libvirtpy - KVM Domain CPU Utilization
+
+
+=head1 CONFIGURATION
+
+Parsed environment variables:
+python-libvirt is required
+
+=head1 LICENSE
+
+GPLv3
+
+SPDX-License-Identifier: GPL-3.0-only
+
+
+=head1 AUTHORS
+
+Julien Rottenberg
+
+Steven Wagner
+
+
+=head1 MAGIC MARKERS
+
+ #%# capabilities=autoconf
+ #%# family=contrib
+
+=cut
+"""
 
 import libvirt
 import sys
 
 conn = libvirt.openReadOnly("qemu:///system")
-if conn == None:
-    print 'Failed to open connection to the hypervisor'
+if conn is None:
+    print('Failed to open connection to the hypervisor')
     sys.exit(1)
 
 try:
     (model, memory, cpus, mhz, nodes, socket, cores, threads) = conn.getInfo()
-except:
-    print 'getInfo failed'
+except BaseException as error:
+    print('getInfo failed: {}'.format(error))
     sys.exit(1)
 
-#print
-#print "KVM running on %d %s %d mhz CPUs w/ %d MB RAM." % (cpus, model, mhz, memory)
-#print
-
 ids = conn.listDomainsID()
-if ids == None or len(ids) == 0:
-    print 'No running domains found.'
+if ids is None or len(ids) == 0:
+    print('No running domains found.')
```

Comment:
`stderr`, please.

Reply:
done.
as I do all the changes in web, without having git installed, I cannot squash them as you suggested (or at least I cannot find a way in web to do it). please can you do again this? Thanks!